ostelco-ios-client/ViewControllers/Home/PurchaseHistoryTableViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  PuchaseHistoryTableViewController.swift
+//  ostelco-ios-client
+//
+//  Created by Prasanth Ullattil on 24/04/2019.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import UIKit
+import Siesta
+import ostelco_core
+
+class PurchaseRecord {
+    let name: String
+    let amount: String
+    let date: String
+
+    init(name: String, amount: String, date: String) {
+        self.name = name
+        self.amount = amount
+        self.date = date
+    }
+}
+
+class PurchaseHistoryTableViewCell: UITableViewCell {
+    @IBOutlet fileprivate weak var date: UILabel!
+    @IBOutlet fileprivate weak var name: UILabel!
+    @IBOutlet fileprivate weak var price: UILabel!
+}
+
+class PucrhaseHistoryTableViewController: UITableViewController {
+    private var records: [PurchaseRecord] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.tableView.tableFooterView = UIView()
+        let refreshControl = UIRefreshControl()
+        refreshControl.addTarget(self, action: #selector(didPullToRefresh), for: .valueChanged)
+        tableView.refreshControl = refreshControl
+        didPullToRefresh()
+    }
+
+    @objc func didPullToRefresh() {
+        getPurchases { purchases, error in
+            self.tableView.refreshControl?.endRefreshing()
+            if let error = error {
+                print("error fetching purchases \(error)")
```

Comment:
a) can you please use `debugPrint` for anything you intend to be a debugging statement?
b) Is there anything we can/should show users if this fails? 
c) Do we want to update the records and reload the data if an error occurs, or only if one doesn't occur?